### PR TITLE
Task_1: try python essentials 

### PR DESCRIPTION
**Pull request by SyrenTen**

### CODE REVIEW

File: task1.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+print("Multi-paradigm programming language:Task 1")
+print("Maslova Vitaliia, i dont know")
+s = ("Vladislav Valeriyovich Ovsyannikov, " * 45)
+a = len(s)
+print(s[:a - 2])
```
Comment (vloooo) [inline task1.py]: in python you can apply negative indexes for slicing.
for ex. -1 is the last item. -2 the item before the last one and etc,
so, you don't need to calculate len() - 2, can just leave -2

File: task1.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+print("Multi-paradigm programming language:Task 1")
```
Comment (vloooo) [inline task1.py]: use single quotes for string literals

Comment (vloooo) [reply task1.py]: I would recommend you to divide your code for logical blocks by blank lines.
for instance in such code you have logical block where you print information about yourself.
the other one is block with printing teacher name.
and the last one is math calculation

**Comment by vloooo** [comment]
your code looks quite cute

**Comment by vloooo** [comment]
please follow a convention in PR's title:
task_{num}: {quite short description of the task}
As well, branch naming convention:
feature/task_{num}

File: task1.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+print('Multi-paradigm programming language:Task 1')
+
+print('Maslova Vitaliia, i dont know')
+
+s = ("Vladislav Valeriyovich Ovsyannikov, " * 45)
```
Comment (vloooo) [inline task1.py]: please use single quotes

File: task1.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+print('Multi-paradigm programming language:Task 1')
```
Comment (vloooo) [inline task1.py:1]: the code looks beautiful, isn't it?

Comment (SyrenTen) [reply task1.py:1]: actually yes

**Comment by SyrenTen** [comment]
Can i click "Merge" for 1 task?

**Review by vloooo — APPROVED** [review]

**Comment by vloooo** [comment]
please merge it

**Pull request merged**